files/faban/src/cloud/benchflow/collectors/faban.go

```diff
@@ -67,9 +67,9 @@ func backupHandler(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		panic(err)
 		} 
-	minioKey := minio.GenerateKey(path)
+	minioKey := minio.GenerateKey("faban")
```

Comment:
the generateKey function takes as argument what will end up being the end of the minio key (the filename in several of our cases). This will generate a key with the same hash for the data collected here regardless of what I put there. I just wrote faban because the path wasn't necessary, it would have made the key longer and more confusing.